packages/react/package.json

```diff
@@ -119,12 +111,6 @@
       "import": "{ Authenticator }",
       "limit": "120 kB"
     },
-    {
```

Comment:
We need to update our canaries to run the size check on the new packages. Have we done that?

Reply:
Will have a PR to `main` for that